src/stock-adjustment-mobile/add-products-page/add-product-page.jsx

```diff
@@ -44,22 +46,21 @@ const AddProductsPage = ({ adjustmentType, appendToAdjustment }) => {
 
     const menu = document.getElementsByClassName("header ng-scope")[0];
     menu.style.display = "none";
-    
-    const CREDIT = "CREDIT";
-    const ISSUE = "Issue";
 
     const decorator = useMemo(() => createDecorator({
         field: /product/,
         updates: {
             stockOnHand: (productVal, itemsVal) => {
                 const orderable = itemsVal.items[0]?.product ?? [];
-                if (itemsVal.items[0].hasOwnProperty('lot')) {
-                    let copiedItemData = Object.assign({}, itemsVal.items[0]);
-                    delete copiedItemData.lot;
-                    itemsVal.items = update(itemsVal.items, { [0] : {$set: copiedItemData} });
-                } 
                 const lotCode = null; 
                 return getStockOnHand(orderable, lotCode);
+            },
+            items: (productVal, itemsVal) => {
+                let newItemsVal = Object.assign({}, itemsVal);
```

Comment:
Should work with `const`. Also, you might use a simpler [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#spread_in_object_literals) instead of `Object.assign`.

Reply:
ok, I will check this with 'const'